node/executor/src/lib.rs

```diff
@@ -465,7 +455,7 @@ mod tests {
 				},
 				CheckedExtrinsic {
 					signed: Some((alice(), 0)),
-					function: Call::Consensus(consensus::Call::remark(vec![0; 120000])),
+					function: Call::Consensus(consensus::Call::remark(vec![0; 403])),
```

Comment:
cc @Kianenigma If you change this 403 to a 402, then two more tests fail regarding big block imports. Basically, I've gathered that if the block is big enough, the test, which is supposed to succeed, fails. If the block is too small, the tests, which are supposed to fail, succeed.

Reply:
perhaps the fee is too big now?

Reply:
Indeed and --at least this matter-- was fixed yesterday. Alice and Bob should just have more balance initially.